### PR TITLE
feat: tls

### PR DESCRIPTION
**Pull request by SSebo**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

support tls connect

### CODE REVIEW

**Comment by PsiACE** [comment]
hi @SSebo , I've made some fixes in #33 to make the clickhouse and mysql tests pass without a hitch.

You can build this branch on the latest commit and then fix the license checker ci.

File: .github/licenserc.yaml [hunk 1]
```diff
@@ -14,3 +14,5 @@ header:
     - "**/*.yaml"
     - "**/*.toml"
     - "**/*.lock"
+    - "**/*.cert"
```
Comment (PsiACE) [inline .github/licenserc.yaml:17]:
```suggestion
    - "**/*.crt"
```

I notice that the new file name is `server.crt`

**Comment by SSebo** [comment]
How about add a integration test case which execute a command `echo "SELECT * FROM foo" | mysql -h 127.0.0.1 --table --ssl-mode=REQUIRE`. but that will required the `mysql` command exist. 
Any good idea?

**Comment by PsiACE** [comment]
> Any good idea?

I think we can add this test to ci. github allows adding commands that do not exist in the environment via package manager.

**Review by PsiACE — APPROVED** [review]
LGTM

**Review by sundy-li — APPROVED** [review]
Great feature!

**Pull request merged**